src/OSPSuite.Core/Domain/Services/ConcurrencyManager.cs

```diff
@@ -23,17 +23,20 @@ public interface IConcurrencyManager
 
    public class ConcurrencyManager : IConcurrencyManager
    {
-      private readonly ICoreUserSettings _coreUserSettings;
-
+      //private readonly ICoreUserSettings _coreUserSettings;
+      private int _maximumNumberOfCoresToUse = 2;
+      //TODO: Check with Michael why ICoreUserSettings is not available through container on Core
+      /*
       public ConcurrencyManager(ICoreUserSettings coreUserSettings)
```

Comment:
ICoreUserSettings is registered via the application. It does not make much sense to use sthg like this in R because we have no such settings. Probably this should not be a dependencies from the ConcurrencyManager?

Reply:
Ok, should we then have an extra class to handle Environment.ProcessorCount on R? Should we just accept what the user sends regardless the number of physical processors?